src/libs/actions/Policy.js

```diff
@@ -1011,6 +1011,10 @@ function openWorkspaceInvitePage(policyID, clientMemberEmails) {
     });
 }
 
+function setWorkspaceInviteMembersDraft(policyID, memberEmails) {
+    Onyx.set(`${ONYXKEYS.COLLECTION.WORKSPACE_INVITE_MEMBERS_DRAFT}${policyID}`, memberEmails);
```

Comment:
We want to replace the existing memberEmails with the new memberEmails entirely since each time user goes to the invite flow, they can select entirely new list of members.